source/documentation/standards/hosting.md.erb

```diff
@@ -0,0 +1,66 @@
+---
+title: How to host services
+last_reviewed_on: 2020-09-01
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+At MOJ you should use the [Cloud
```

Comment:
If we want to set the precedence of using the Cloud Platform as the default (and not using a separate AWS account, for example), would it be better to change "should" to "must"?